css/main.css

```diff
@@ -1,3 +1,254 @@
-/* Add your custom CSS here */
+* {
+  box-sizing:border-box;
+  margin:0 ;
+  padding: 0 0;
+}
+.hamIcon{
+  position: absolute;
+  right: 10px;
+  top: 20px;
+}
 
-/* Dont' forget to link this file to your HTML in the <head> */
+.md-col-12 {
+    width: 100%;
+ }
+.header_customize{
+  background-color:#466381;
+  display:flex;
+  justify-content:space-between;
+  font-family: 'Vast Shadow', cursive;
+}
+.col-12 {
+  width: 100%;
+}
+.col-8 {
+  width: 66.666666%;
+}
+.col-6 {
+  width: 50%;
+}
+.col-10 {
+  width: 83.333333%;
+}
+.col-3 {
+  width: 25%;
+}
+.col-2 {
+  width: 16.666666%;
+}
+.col-1 {
+  width: 8.333333%;
+}
```

Comment:
You don't need to write this - that's what grid.css is for! It provides all the grid classes pre-written for you. You just need to import it in the `<head>` of your index.html and they will be ready to use!